http://bugzilla.xfce.org/show_bug.cgi?id=6613

xfce4_cellmodem_plugin-pin_helper.o: In function `pin_helper_pin_entry_callback':
pin_helper.c:(.text+0xa6): undefined reference to `xfce_warn'

--- configure.ac
+++ configure.ac
@@ -18,6 +18,8 @@
 AC_CHECK_FUNCS([vasprintf])
 
 XDT_CHECK_PACKAGE([LIBXFCE4PANEL], [libxfce4panel-1.0], [4.3.20])
+XDT_CHECK_PACKAGE([LIBXFCEGUI4], [libxfcegui4-1.0], [4.3.20])
+
 XDT_I18N([cs pt])
 
 AC_ARG_ENABLE([debug],
--- panel-plugin/Makefile.am
+++ panel-plugin/Makefile.am
@@ -3,10 +3,12 @@
 
 xfce4_cellmodem_plugin_CFLAGS = \
         -DPACKAGE_LOCALE_DIR=\"$(localedir)\" \
-	@LIBXFCE4PANEL_CFLAGS@
+	@LIBXFCE4PANEL_CFLAGS@ \
+	@LIBXFCEGUI4_CFLAGS@
 
 xfce4_cellmodem_plugin_LDADD = \
-	@LIBXFCE4PANEL_LIBS@ 
+	@LIBXFCE4PANEL_LIBS@ \
+	@LIBXFCEGUI4_LIBS@
 
 xfce4_cellmodem_plugin_SOURCES = \
 	cellmodem.c modem_driver.c \
